src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -379,6 +379,18 @@ private static unsafe string ReadGuid(Table.TableValueHolder entryHolder)
             return guid;
         }
 
+        public bool ContainsGuid(ClusterOperationContext context, string guid)
```

Comment:
Guid is a specific term, but you are passing just ids here, which can be any strings

Reply:
Okay, so would something like: `CommandUniqueId` would be a more proper term?

Reply:
Yes